fastlane_core/spec/project_spec.rb

```diff
@@ -419,7 +419,7 @@ def count_processes(text)
       end
 
       it "runs simple commands" do
-        cmd = 'echo "HO"'
+        cmd = 'echo HO'
```

Comment:
Part of this integration test was probably to check that running the program with those quotes worked as the platform is was supposed to be supported on. Right?

Reply:
The test name or expect doesn't indicate that the quotes were relevant. I don't think this was meant to test the `echo` implementation (which is what is different on Windows and macOS), but that `run_command` indeed runs a command in the expected way, right?

Alternatively to my change, we could also change the `.to eq()` to `should include()` to achieve the same thing. Would that be better?

Reply:
echo behaves differently on Windows and Unix. Let's keep that test testing running programs only, and if needed we will add per platform tests that check quoting. Maybe add a comment that the command is written in a way that it works the same on all platforms.

Reply:
added the comment